CHANGELOG.md

```diff
@@ -3,14 +3,47 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0dev - [date]
+## v1.0 - UNSC Infinity - [2023-10-02]
 
 Initial release of sanger-tol/curationpretext, created with the [sager-tol](https://nf-co.re/) template.
 
 ### `Added`
 
-### `Fixed`
+- Subworkflow to generate tracks containing telomeric sites.
+- Subworkflow to generate Pretext maps and images
+- Subworkflow to generate repeat density tracks.
+- Subworkflow to generate longread coverage tracks from pacbio data.
+- Subworkflow to generate gap tracks.
 
-### `Dependencies`
+### Parameters
 
```

Comment:
List all parameters in new version, similar to how you are doing for software dependencies.

Reply:
Done